### DISCUSSION TITLE
Error: AnimatedValue: Attempting to set value to undefined

### DISCUSSION THREAD

**Discussion by vinodsnayvik** [react-native-community/react-native-tab-view#1390] [Q&A]
I have added this package for showing tab view in the middle of screen. I followed same procedure which is available in readme file. 
I got an error that  
```
AnimatedValue: Attempting to set value to undefined
```
May be this is common error but i am not able to solve it. When i remove TabView component then remaining code is working.
Could someone tell me what is issue ?

**Comment by okwasniewski** [comment]
Hey @vinodsnayvik,

Can you create a snack reproducing your issue / post a code snippet? It will be easier to help you with that.
